### ISSUE TITLE
Story Arc Management - Status not updating

### ISSUE THREAD

**Issue by rmgy733** [evilhero/mylar#1146]
I recently started using Story Arc Management. Everything works great -- Issues are downloaded from NZBs, numbered, and added to the appropriate StoryArc folder. 

The only issue I am having is the Story Arc status for each individual comic remains as "Snatched" unless I press the "Search for Watchlist Matches" button, at that point all "Snatched" items will show as "Downloaded" as expected.

Edit: I'm using the latest development branch (atleast according to the Mylar update checker) - Version: 60140e8c900ae6c2acc3c9bf319506b6514fbed4 (development) and Python 2.7.10.

**Comment by evilhero** [comment]
It's very possible it's not correctly finding the given issue in the arcs. 

Can I ask how it is that you're grabbing the arc issues?  

Are you marking them as one-off, or downloading them as a warchlist item and then letting mylar determine if it belongs to an arc? Are you choosing the Search for Missing option on the arc details page?

Alot of how mylar handles arc issues is dependant on how the issue was initially grabbed. 

I would guess based on your description that the issues are being snatched as normal watchlist items and post-processed but the status isn't being updated thereafter?

**Comment by rmgy733** [comment]
Heres a step by step method of how I am using Story Arcs:

1. Manage -> Advanced Options -> Story Arc Management
2. On the Options -> Add Story Arc area I search for a series and then press "+ Add this Story Arc" on the results page.
3. This brings me to the Story Arc page. Issues that I have already as Watchlist items are automatically registered as "Downloaded"
4. I click "Search For Missing" - The Status either changes to Wanted or Snatched for each non-Downloaded comic in the Arc.
5. The "Snatched" items are downloaded/processed and moved to the /StoryArcs/<storyarcname>/ folder and numbered properly, however the status of the comic itself remains as "Snatched" on the Story Arc view. It doesn't recognize that the comic has been downloaded/processes to the /StoryArcs/ folder. 

Hopefully this helps clarify :). Basically everything works as expected  -- the comics are being processed to the StoryArc folder, but for whatever reason the StoryArc view is still showing the comic as "Snatched" rather then "Downloaded".

**Comment by evilhero** [comment]
Maybe I'm doing something wrong on this end (very likely when trying to duplicate someone's problems) - but I tried to do a 'Search for Missing' on a Story-Arc. They got snatched, downloaded, and  the status' did end up getting changed to Downlaoded Status at the end of it all. The only thing I did notice is that I had to refresh the browser page itself as the page wasn't updated - once I refreshed the page, the status' were shown to be in the proper status. The same thing would occur if I left the given arc page, and then went back to it - the status' would be updated properly.

I'm assuming the arc issues you were downloading weren't in your watchlist already - as whether or not they exist on the watchlist plays a factor in how Mylar handles the post-processing afterwards.
